### PR TITLE
boilerplate index.html

### PR DESCRIPTION
**Pull request by simonabali**
Please review.

### CODE REVIEW

**Comment by jovandyaz** [comment]
Listouh...

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Git Collab Project</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+
```
Comment (jovandyaz) [inline dist/index.html:11]: code review!

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Git Collab Project</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
```
Comment (LeahChercham) [inline dist/index.html:10]: Add some more content !

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+    <script src= "main.js"> </script> 
+</body>
+</html>
```
Comment (fnando-gomez) [inline dist/index.html:14]: If we finish this Tomer will cook some tacos.

File: dist/index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```
Comment (Tomshsh) [inline dist/index.html:6]: imagine morgan freeman reads this code to you!

**Review by jovandyaz — COMMENTED** [review]
I did a review Simo.
;)

**Pull request merged**